nimbleparse/src/main.rs

```diff
@@ -187,6 +189,11 @@ fn main() {
         }
     };
 
+    if dump_state_graph {
+        println!("Stategraph:\n{}\n", sgraph.pp_core_states(&grm));
+        process::exit(0);
```

Comment:
My suggestion is that we don't `exit` here: we print out the stategraph and continue. That simplifies the argument parsing in the sense that users *always* have to provide `.l`, `.y`, and input files. We could be more clever, but documenting that in a digestible way becomes a pain, and I think the simple approach is adequate.

Reply:
Yeah the problem with my testing was that I didn't actually have valid input for any of the github provided parsers I was testing, while it is likely that /dev/null would be a valid for most perhaps even all of them, it was easier to just avoid complications with input/AST production.

Perhaps that is a special case that is no longer necessary, and I suppose it is easy to add the `exit` call if I need something like that again?

Reply:
That could be good, there were a couple of actually the most heavily used libraries depending upon grmtools
which I couldn't test with this approach because they used a manually implemented lexer.

Reply:
I think this PR is useful (without the `exit`) and worth keeping, but an additional `lrtable` binary PR would be cool too!

Reply:
Should be fixed in 2a13f24

There was a little more to it, also needed to avoid duplicate printing of the state graph when there are conflicts.